shared/src/main/scala/coop/rchain/catscontrib/SyncInstances.scala

```diff
@@ -29,16 +30,24 @@ object SyncInstances {
 
       def bracketCase[A, B](
           acquire: Effect[A]
-      )(use: A => Effect[B])(release: (A, ExitCase[Throwable]) => Effect[Unit]): Effect[B] =
-        acquire flatMap { state =>
-          try {
-            use(state)
-          } catch {
-            case t: Throwable => raiseError(t)
-          } finally {
-            release(state, ExitCase.Completed)
-          }
-        }
+      )(use: A => Effect[B])(
```

Comment:
why do we even have our own instance of `Sync` for `type Effect[A] = EitherT[Task, E, A]`?

Monix has a derivation rule for that (not sure or how long):

```
//object Sync

  implicit def catsEitherTSync[F[_]: Sync, L]: Sync[EitherT[F, L, ?]] =
    new EitherTSync[F, L] { def F = Sync[F] }
```

Task has Sync IIRC. Let's get rid of this just use the cats instance.

Reply:
makes sense!

Reply:
https://github.com/rchain/rchain/pull/1902